### PR TITLE
Bugbuster buster

### PR DESCRIPTION
**Pull request by danielgustafsson**
This PR isn't terribly exciting to be honest, mainly removing dead and superflous test suites. I got a report that a patch of mine failed the Bugbuster oid_consistency test and based on that I followed the rabbit down the hole. The oid_consistency test is addressed in #1709, this PR contains some Bugbuster cleanup that fell out when looking at that one.

The individual commit messages contain more information for reviewing.

### CODE REVIEW

**Comment by ashwinstar** [comment]
How about instead move what's good remaining (sql and and files) in there we run to ICW and kill the full folder ?

**Comment by danielgustafsson** [comment]
I was contemplating doing that, but I figured that doing so requires figuring out what is actually good.. since if we don't, whatever we pull over wont ever be looked at once in ICW and will just "cost time". I unfortunately think we need to pull selectively.

File: src/test/regress/bugbuster/sql/security.sql [hunk 1]
```diff
@@ -1,64 +0,0 @@
--- ***** From TIME_BASED_CONSTRAINT suite *****
-CREATE USER invalidUser1 WITH LOGIN DENY BETWEEN DAY 'Monday' TIME '00:00' AND DAY 'Monday' TIME '25:00';
```
Comment (hlinnaka) [inline src/test/regress/bugbuster/sql/security.sql]: I don't see tests exactly like this in 'role' or 'auth_constraint'. There are some tests with English weekdays, like 'Tuesday', but I don't see an invalid TIME like this there. Not sure if the other DENY BETWEEN variants below are covered either.

**Comment by hlinnaka** [comment]
+1, but please double-check the coverage of the 'security' test, per my earlier comment.

File: src/test/regress/bugbuster/sql/security.sql (hunk 1 above)
Comment (danielgustafsson) [inline src/test/regress/bugbuster/sql/security.sql]: Nice catch, I had confused myself after all that test file staring so missed these. Rebased and updated the PR with the ones I felt were of interest ported to auth_constraint.

**Comment by danielgustafsson** [comment]
Pushed to master after a green PR pipeline run, thanks for review!

**Pull request closed without merging**